idl/partition_checksum.idl.hh

```diff
@@ -110,6 +110,8 @@ struct node_ops_cmd_request {
     std::unordered_map<gms::inet_address, std::list<dht::token>> bootstrap_nodes;
     // Optional field, list uuids of tables being repaired, set by repair cmd
     std::list<table_id> repair_tables;
+    // Optional field, list of hosts to quarantine when done
+    std::unordered_map<locator::host_id, db_clock::time_point> hosts_to_quarantine;
```

Comment:
Sending an empty map will not break stuff if the other side is not yet upgraded?

(I vaguely recall it's fine to just send anything if we append fields to a struct, because the other side will simply truncate it if it doesn't know about the last field, but just making sure)

Reply:
No, it shouldn't.
RPC messages can be extended if their prefix is backward compatible